Examples/Google/SpeechToText/Sources/Launch/AppDelegate.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import GRPC
+import Logging
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+  
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
+    
+    LoggingSystem.bootstrap {
+      var handler = StreamLogHandler.standardOutput(label: $0)
+      handler.logLevel = .debug
```

Comment:
Do we actually want debug logging?

Reply:
As someone who is still relatively new to gRPC Swift, I found it helpful to see the logs and wrap my head around what was happening. 

I see the other example projects have the level set to `.critical` logging! I'll change it to be consistent. 👍